template/amber/index.amber

```diff
@@ -0,0 +1,46 @@
+extends base
+
+block append head
+    title Active Repositories
+
+block header
+    ol
+        li Active Repositories
+
+    ul.nav.nav-tabs
+        li.nav-item
+            a.nav-link.active[href="/"] Active Repositories
+        li.nav-item
+            a.nav-link[href="/repos"] My Repositories
+
+block content
+    div.container
+        if len(Repos) == 0
+            div.alert.alert-info
+                | Your repository list is empty.
+
+        else
+            div.row.repo-row
+                input.repo-search[type="search"][placeholder="Filter..."]
+                div.repo-list
+                    each $repo in Repos
+                        div.col-sm-4
+                            a.card[href="/"+$repo.FullName]
+                                div.card-header
+                                    if $repo.Avatar != ""
+                                        img.avatar[src=$repo.Avatar]
+                                    else
+                                        img.avatar[src="/static/images/dummy.png"]
+                                div.card-block
+                                    h3.login #{$repo.Name}
+                                    div.full_name.hidden #{$repo.FullName}
+block append scripts
+    if len(Repos) != 0
+        script
+            $(window).load(function(){
+            $(".repo-list").searcher({
```

Comment:
Which indentation is wrong? I did also add a new level of indentation so the filter box doesn't show up when the repository lists are empty since there's nothing to filter on.